cmd/tailscale/cli/bugreport.go

```diff
@@ -16,6 +17,15 @@ var bugReportCmd = &ffcli.Command{
 	Exec:       runBugReport,
 	ShortHelp:  "Print a shareable identifier to help diagnose issues",
 	ShortUsage: "bugreport [note]",
+	FlagSet: (func() *flag.FlagSet {
+		fs := newFlagSet("doctor")
+		fs.BoolVar(&bugReportArgs.doctor, "doctor", false, "run additional in-depth checks")
```

Comment:
diagnose? diag? Or make it a comma-separated list of check names with an "all" alias? `--check=all` ?